README.md

```diff
@@ -1,101 +1,164 @@
-# Portfolio: setup and mobile-first<a name="readme-top"></a>
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
   - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
+  - [Deployment](#deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Portfolio:setup and mobile-first**
-is an introductory project to start building my personal Portifolio.
+# 📖 Portfolio: finish mobile version <a name="about-project"></a>
+
+>This is a personal portfolio, Developed based on a Given template(Templete three)
+
+**Portfolio: finish mobile version** is the next project from setup mobile first portifolio that includes the works and about me contribution.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> css and html
+>The project uses mainly the following technologies:
+
+<details>
+  <summary>CSS</summary>
+</details>
+<details>
+  <summary>HTML</summary>
+</details>
+<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- **Html & css**
-- **Linters config**
-- **Responsive styles**
+> Describe between 1-3 key features of the application.
```

Comment:
- [ ] Delete all lines starting with `>` and are containing the original guiding text  
![inst](https://user-images.githubusercontent.com/99656687/236625636-b5487330-57bb-4d4c-89c6-c90191731c8d.png)

Reply:
Thank you for your view i have corrected all yet

Reply:
- [ ]  Have all lines starting with `>` which contain the guiding texts deleted. Delete all of them from your `readme`
![inst](https://user-images.githubusercontent.com/99656687/236627109-d37845d6-382d-4970-92de-d78e642d170d.png)